Description: Fix refresh errors on Win2012 RDP connections with speed=ADSL
Author: Oleksandr Shneyder <o.shneyder@phoca-gmbh.de>
diff -rupN a/nx-X11/programs/Xserver/hw/nxagent/Render.c b/nx-X11/programs/Xserver/hw/nxagent/GCOps.c
--- a/nx-X11/programs/Xserver/hw/nxagent/GCOps.c	2014-05-06 12:52:35.082288455 +0200
+++ b/nx-X11/programs/Xserver/hw/nxagent/GCOps.c	2014-05-06 12:54:00.258710819 +0200
@@ -339,7 +339,7 @@ FIXME: The popup could be synchronized w
    */
 
   if ((pDstDrawable -> type == DRAWABLE_PIXMAP &&
-          nxagentOption(DeferLevel) > 0) || nxagentOption(DeferLevel) >= 2)
+          nxagentOption(DeferLevel) > 0) || nxagentOption(DeferLevel) >= 3)
   {
     pClipRegion = nxagentCreateRegion(pSrcDrawable, NULL, srcx, srcy,
                                           width, height);
